Sources/CartonCLI/Commands/Bundle.swift

```diff
@@ -42,6 +46,10 @@ struct Bundle: AsyncParsableCommand {
   @Flag(help: "Emit names and DWARF sections in the .wasm file.")
   var debugInfo: Bool = false
 
+  @Option(name: .long,
+          help: "Which optimizations to apply to the .wasm binary output.\nAvailable values: \(WasmOptimizations.allCases.map { $0.rawValue }.joined(separator: ", "))")
```

Comment:
```suggestion
  @Option(
    name: .long,
    help: "Which optimizations to apply to the .wasm binary output.\nAvailable values: \(
      WasmOptimizations.allCases.map { $0.rawValue }.joined(separator: ", ")
    )"
  )
```

Reply:
Hi @MaxDesiatov the code does not build like this. I don't know why but it appears putting `WasmOptimizations.allCases.map { $0.rawValue }.joined(separator: ", ")` onto a new line like this breaks something to do with Swift's parser. I believe it's a Swift bug

Reply:
Actually, it seems that any newline characters within `"`-delimited strings will break the parsing. I'm not sure this is even a bug but may be by design. So I need to revert this change otherwise the code will not compile.

Reply:
I think you'll just have to use `"""`-delimited multiline strings for multiline interpolation to work.